crates/hir/src/lib.rs

```diff
@@ -2195,6 +2200,10 @@ impl Type {
         matches!(self.ty.kind(&Interner), TyKind::Ref(hir_ty::Mutability::Mut, ..))
     }
 
+    pub fn is_reference(&self) -> bool {
+        matches!(self.ty.kind(&Interner), TyKind::Ref(hir_ty::Mutability::Not, ..))
```

Comment:
```suggestion
        matches!(self.ty.kind(&Interner), TyKind::Ref(..))
```
This should ignore the mutability